vertica/datadog_checks/vertica/vertica.py

```diff
@@ -44,7 +45,9 @@ def __init__(self, name, init_config, instances):
         self._timeout = float(self.instance.get('timeout', 10))
         self._tags = self.instance.get('tags', [])
 
-        self._client_lib_log_level = self.instance.get('client_lib_log_level')
+        self._client_lib_log_level = self.instance.get(
+            'client_lib_log_level', logging.getLevelName(self.log.logger.getEffectiveLevel())
+        )
```

Comment:
```suggestion
        self._client_lib_log_level = self.instance.get('client_lib_log_level', self.log.logger.getEffectiveLevel())
```

It might better to do this ^.

In case the effective level is not among the standard values, `logging.getLevelName` might fail.

Reply:
Oh, right, `vertica-python` can accept `logging.*` constants, so `.getEffectiveLevel()` can be passed directly… Good catch again.